tests/manage/nfs_feature/test_nfs_feature_enable_for_ODF_clusters.py

```diff
@@ -0,0 +1,449 @@
+            params=self.nfs_spec_disable,
+            format_type="merge",
+        ), "storagecluster.ocs.openshift.io/ocs-storagecluster not patched"
+
+        # Enable ROOK_CSI_ENABLE_NFS via patch request
+        assert self.rook_ceph_cm_obj.patch(
+            resource_name="rook-ceph-operator-config",
+            params=self.rook_csi_config_disable,
+            format_type="merge",
+        ), "configmap/rook-ceph-operator-config not patched"
+
+        # Delete the nfs StorageClass
+        cmd_delete_nfs_sc = "delete sc " + self.nfs_sc
+        self.storage_cluster_obj.exec_oc_cmd(cmd_delete_nfs_sc)
+
+        # Delete CephNFS
+        cmd_delete_cephnfs = "delete CephNFS ocs-storagecluster-cephnfs"
+        self.storage_cluster_obj.exec_oc_cmd(cmd_delete_cephnfs)
+
+        # Delete ocs nfs Service
+        cmd_delete_nfs_service = "delete service rook-ceph-nfs-my-nfs-load-balancer"
+        self.storage_cluster_obj.exec_oc_cmd(cmd_delete_nfs_service)
+
+        # Delete nfs-ganesha pod
+        self.nfs_ganesha_pod_obj.delete()
+        self.nfs_ganesha_pod_obj.ocp.wait_for_delete(
+            self.nfs_ganesha_pod_obj.name, 180
+        ), f"Pod {self.nfs_ganesha_pod_obj.name} is not deleted"
+
+    def teardown(self):
+        """
+        Delete cephnfs CR,ocs nfs service and nfs storageclass
+        """
+        self.nfs_disable()
+
+    def test_nfs_feature_enable(
+        self,
+        pod_factory,
+    ):
+        """
+        This test is to validate nfs feature enable after deployment of  ODF(4.11) cluster
+
+        Steps:
+        1:- Check cephnfs resources not available by default
+        2:- Enable nfs feature for storage-cluster
+        3:- Check nfs-ganesha server is up and running
+        4:- Check csi-nfsplugin pods are up and running
+
+        """
+        # Checks cephnfs resources not available by default
+        cephnfs_resource = self.storage_cluster_obj.exec_oc_cmd("get cephnfs")
+        if cephnfs_resource is None:
+            log.info(f"No resources found in openshift-storage namespace.")
+
+        # Enable nfs feature for storage-cluster
+        self.nfs_enable()
+
+        # Check nfs-ganesha server is up and running
+        assert self.nfs_ganesha_pod_obj.wait_for_resource(
+            resource_count=1,
+            condition=constants.STATUS_RUNNING,
+            selector="app=rook-ceph-nfs",
+            dont_allow_other_resources=True,
+            timeout=60,
+        )
+
+        # Check csi-nfsplugin and csi-nfsplugin-provisioner pods are up and running
+        assert self.nfs_plugin_pod_obj.wait_for_resource(
+            resource_count=self.num_of_cephfsplugin_pods,
+            condition=constants.STATUS_RUNNING,
+            selector="app=csi-nfsplugin",
+            dont_allow_other_resources=True,
+            timeout=60,
+        )
+
+        assert self.nfs_plugin_provisioner_pod_obj.wait_for_resource(
+            resource_count=self.num_of_cephfsplugin_provisioner_pods,
```

Comment:
`num_of_cephfsplugin_provisioner_pods` ?

Reply:
Addressed in latest commit.